src/Data/Quote/LocalStorage/localStorageQuoteDataClass.ts

```diff
@@ -689,6 +696,13 @@ async function attributes(): Promise<DataClassSchema> {
       { title: lang === 'de' ? 'Vertriebspartner' : 'Sales partner' },
     ],
     status,
+    technicalAgent: [
+      'reference',
+      {
+        title: lang === 'de' ? 'Technischer Bearbeiter' : 'Techincal agent',
```

Comment:
* [ ] Typo "Technical"

Reply:
same fix as above

Reply:
`GET https://web102.crm.pisasales.de/portal/quote/schema` still reports the wrong value:

```json
       "technicalAgent": [
            "reference",
            {
                "title": "Techincal agent",
                "to": "User"
            }
        ],
```